paperclip.gemspec

```diff
@@ -35,7 +35,7 @@ Gem::Specification.new do |s|
   s.add_development_dependency('rspec', '~> 3.0')
   s.add_development_dependency('appraisal')
   s.add_development_dependency('mocha')
-  s.add_development_dependency('aws-sdk', '>= 2.0.34', '< 3.0')
+  s.add_development_dependency('aws-sdk', '>= 2.3.0', '< 3.0')
```

Comment:
Is this bump necessary? I wouldn't require it to everyone; the exception takes care of it for us. CI should anyway install latest version, as that's what we are specifying here.